presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,212 @@
+    authentication.ldap.group-dn=CN=Group,OU=America,DC=corp,DC=domain,DC=com
+    authentication.ldap.user-object-class=person
+
+    http-server.https.enabled=true
+    http-server.https.port=8443
+
+    http-server.https.keystore.path=/etc/presto_keystore.jks
+    http-server.https.keystore.key=keystore_password
+
+======================================================= ======================================================
+Property                                                Description
+======================================================= ======================================================
+``authentication.ldap.enabled``                         Enable LDAP Authentication for the Presto coordinator.
+                                                        Must be set to ``true``.
+``authentication.ldap.server-type``                     The type of LDAP Server Implementation.
+                                                        Set to ``active_directory`` or ``openLDAP``.
+``authentication.ldap.url``                             The url to the LDAP Server. We recommend using 
+                                                        ``ldaps://`` to enable Secure LDAP over SSL.
+``authentication.ldap.ad-domain``                       The domain name of the Active Directory server.
+                                                        This property is optional if 
+                                                        ``authentication.ldap.server-type`` is set to
+                                                        ``openLDAP``. Eg: ``corp.domain.com``.
+``authentication.ldap.base-dn``                         The base LDAP distinguished name for the user who
+                                                        who tries to connect to the server. This property is 
+                                                        optional if ``authentication.ldap.server-type`` is set
+                                                        to ``active_directory``, but mandatory if
+                                                        ``authentication.ldap.group-dn`` is set.
+``http-server.https.enabled``                           Enables HTTPS access for the Presto coordinator.
+                                                        Should be set to ``true``.
+``http-server.https.port``                              HTTPS server port.
+``http-server.https.keystore.path``                     The location of the Java Keystore file that will be
+                                                        used to secure TLS.
+``http-server.https.keystore.key``                      The password for the keystore. This must match the
+                                                        password you specified when creating the keystore.
+======================================================= ======================================================
+
+Group Based Authorization Support
```

Comment:
its authorization but based on LDAP. Renamed to "Authorization based on LDAP group membership"